code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1133,7 +1132,7 @@
 		/mob/living/simple_animal/hostile/mimic/crate,
 		/mob/living/simple_animal/hostile/mimic/crate/chest,
 		/mob/living/simple_animal/hostile/mimic/crate/item,
-		) + typesof(/mob/living/simple_animal/hostile/humanoid) + typesof(/mob/living/simple_animal/hostile/asteroid) //Exclusion list for things you don't want the reaction to create.
```

Comment:
well they're still being picked at random from a large pool of mobs, you'd have to pop a few of those to roll one and people don't commonly weaponize the hostile reaction. If someone wants a chance to get the cool things they'll have to pop a few gold cores and plasma fire the whole thing behind a shield